pkg/crypto/api.go

```diff
@@ -37,4 +39,10 @@ type Crypto interface {
 	// VerifyMAC determines if mac is a correct authentication code (MAC) for data
 	// using a matching MAC primitive in kh key handle and returns nil if so, otherwise it returns an error.
 	VerifyMAC(mac, data []byte, kh interface{}) error
+
+	// WrapKey will execute key wrapping of cek using apu, apv and recipient public key found in kh.
+	WrapKey(cek, apu, apv []byte, kh interface{}) (*composite.RecipientWrappedKey, error)
+
+	// UnWrapKey unwraps a key in recWK using recipient private key kh.
+	UnWrapKey(recWK *composite.RecipientWrappedKey, kh interface{}) ([]byte, error)
```

Comment:
Unwrap is a single word.

Reply:
my editor gave a spelling mistake for Unwrap ..

Reply:
updated

Reply:
@Baha-sk Now that you mention it, I think I saw that happen in my local GoLand at some point as well...